chain/client/src/client.rs

```diff
@@ -1673,14 +1673,15 @@ impl Client {
                     shard_id,
                     is_forwarded
```

Comment:
let's modify this debug log -- as we no longer record transaction if this node is not a validator.
(we should probably do a 3 debugs in line 1683, 1689 and 1693 .